apps/io/feeding_service_webhook.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2016 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+from superdesk.resource import Resource
+from superdesk.services import BaseService
+from superdesk.io.commands import update_ingest
+from superdesk.io.registry import registered_webhooks
+import superdesk
+
+
+class FeedingServiceWebhookResource(Resource):
+    public_methods = ['GET', 'POST']
```

Comment:
or maybe really use this `a handshake parameter` but HTTPS should be used for sure in this case...